src/extensions/sb-ncbr/partial-charges/behavior.ts

```diff
@@ -0,0 +1,38 @@
+import { LociLabelProvider } from '../../../mol-plugin-state/manager/loci-label';
```

Comment:
Can you please add `readme.md` file here with some basic documentation about what this extension does and how it works?